src/guest/start.c

```diff
@@ -44,7 +44,7 @@ static const char *fixed_cmd =
 	" -device usb-mouse"
 	" -device usb-kbd"
 	" -device intel-hda -device hda-duplex"
-	" -audiodev id=android_spk,timer-period=5000,driver=pa,in.fixed-settings=off,out.fixed-settings=off"
+	" -audiodev id=android_spk,timer-period=5000,driver=pa,in.latency=50000,out.latency=50000"
```

Comment:
please also check the sample rate settings. previously my experience is 48000k maybe better.

Reply:
@lchen43 Now also it is 48k
cat /proc/asound/card0/pcm0p/sub0/hw_params
access: RW_INTERLEAVED
format: S16_LE
subformat: STD
channels: 2
rate: 48000 (48000/1)
period_size: 4080
buffer_size: 16320